api_tests/e2e/rfc003/eth-erc20_btc/test.js

```diff
@@ -312,9 +319,9 @@ describe("RFC003: ERC20 for Bitcoin", () => {
             .request(alice.comit_node_url())
             .get(
                 alice_redeem_href +
-                "?address=" +
-                alice_final_address +
-                "&fee_per_byte=20"
+                    "?address=" +
```

Comment:
❓did you format this manually ?

Reply:
Nope, that's just `prettier` :nail_care: .